vcproj/canary.vcxproj

```diff
@@ -288,6 +291,7 @@
     <ClCompile Include="..\src\lua\functions\core\game\game_functions.cpp" />
     <ClCompile Include="..\src\lua\functions\core\game\global_functions.cpp" />
     <ClCompile Include="..\src\lua\functions\core\game\bank_functions.cpp" />
+    <ClCompile Include="..\src\lua\functions\core\game\zone_functions.cpp" />
```

Comment:
I believe those changes in vcxproj are in the wrong commit.

Reply:
squashed into the wrong one, eh, too late now